packages/core/src/lib/toast/toast-banner.svelte

```diff
@@ -13,41 +15,80 @@ export let dismiss: () => void;
 export let action: { text: string; handler: () => unknown } | undefined =
   undefined;
 
+/** The severity of the notification you want to show users*/
+export let variant: ToastVariantType = 'success';
+
 /** Additional CSS classes to pass to the banner. */
 let extraClasses: cx.Argument = '';
 export { extraClasses as cx };
+
+let icon: IconName | null = null;
```

Comment:
We generally avoid `null`; `undefined` is more predictable. See [this article](https://github.com/sindresorhus/meta/discussions/7) for background